drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -46,7 +52,11 @@
         </div>
         <div class="row">
             <label>{{i18n "Background image URL (alternative to the text)"}}</label>
-            <textarea class="background-image">{{ backgroundImage }}</textarea>
+            <textarea class="background-image-url">{{ backgroundImage }}</textarea>
```

Comment:
Can you please convert this to an `<input type="text"` while you're at this? I don't think it should be a `textarea`.